.eslintignore

```diff
@@ -0,0 +1,2 @@
+!/.storybook
+!/.jest
```

Comment:
Oh, that's so cool, I didn't know we can use that pattern!

Reply:
jap thats nice, found it in a GH issue was the simples syntax to include them